cltk/corpus/latin/corpora.py

```diff
@@ -86,5 +86,9 @@
     {'location': 'remote',
      'type': 'text',
      'name': 'latin_text_poeti_ditalia',
-     'origin': 'https://github.com/cltk/latin_text_poeti_ditalia.git'}
+     'origin': 'https://github.com/cltk/latin_text_poeti_ditalia.git'},
+    {'location': 'remote',
+     'type': 'text',
+     'name': 'latin_text_tesserae_collection',
+     'origin': 'https://github.com/cltk/latin_text_tesserae_collection.git'}
```

Comment:
Thank you for adding a new corpora, this is great!

Reply:
Thanks Todd for all the helpful pointers.

James, I have bandwidth to work on this, so will push out as much as possible today. Let's focus on resolving the particular points, first.

To this one, I see you still have [the Latin Tesserae repo](https://github.com/jamesgawley/latin_text_tesserae_collection) under your account. Before transferring it to the cltk org, we need you to add a few things:
* Source: Be as specific as you can about where these texts were found and how gathered.
* Licensing: What FOSS license is used by the Tesserae project?